apps/ledger-live-mobile/src/components/WebPTXPlayer/index.tsx

```diff
@@ -185,6 +191,7 @@ export const WebPTXPlayer = ({ manifest, inputs }: Props) => {
         inputs={inputs}
         onStateChange={setWebviewState}
       />
+      {isLoading ? <Loading /> : null}
```

Comment:
You can just use `webviewState.loading` IMO
```suggestion
      {webviewState.loading ? <Loading /> : null}
```

Reply:
@cpellizzari-ledger this is what you were talking about - thanks!